src/test/java/com/salesforce/dynamodbv2/testsupport/DocGeneratorRunner.java

```diff
@@ -543,7 +544,12 @@ private String buildTableName(int ordinal) {
         }
 
         private Path getOutputFile(String outputFilePath) {
-            new File(outputFilePath).getParentFile().mkdirs();
+            File parentDir = new File(outputFilePath).getParentFile();
```

Comment:
`parentDir` could be preceded by `final`

Reply:
addressed in https://github.com/salesforce/mt-dynamo/pull/332/commits/92a7aa5ea7083c3bb22686455f7c645107fcb249